acados/ocp_nlp/ocp_nlp_common.c

```diff
@@ -1106,6 +1107,9 @@ void ocp_nlp_opts_initialize_default(void *config_, void *dims_, void *opts_)
     // globalization
     opts->alpha_min = 0.05;
     opts->alpha_reduction = 0.7;
+    opts->full_step_dual = 0;
+    opts->line_search_use_sufficient_descent = 0;
```

Comment:
Can you explain somewhere what this option does?

Reply:
I added and documented the options to the Python interface.
https://github.com/acados/acados/blob/8d2766b4ebaa321e55e10ad8b818bf0dc991d6f7/interfaces/acados_template/acados_template/acados_ocp.py#L2374-L2409
The docstrings there are used to generate the documentation on the website.